src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -109,7 +105,12 @@ void CalculatorProgrammerRadixOperators::BitshiftFlyout_Checked(Platform::Object
         announcementString = m_rotateCarryShiftButtonContent;
     }
 
-    this->BitShiftFlyout->Hide();
+    if (announcementString != m_selectedShiftButtonContent)
```

Comment:
I like the new behavior of not closing when the same button is selected. 👍

Reply:
thanks!